esm-samples/jsapi-deno/request.ts

```diff
@@ -1,7 +1,7 @@
 // @deno-types="npm:@arcgis/core/interfaces.d.ts"
-import { Application, Context } from "https://deno.land/x/oak@v11.1.0/mod.ts";
-import config from "npm:@arcgis/core/config.js";
-import esriRequest from "npm:@arcgis/core/request.js";
+import { Application, Context } from "https://deno.land/x/oak@v12.5.0/mod.ts";
+import config from "npm:@arcgis/core@4.27.0/config.js";
+import esriRequest from "npm:@arcgis/core@4.27.0/request.js";
```

Comment:
Do these work without `.0` so they'll get the patches?

Reply:
Good catch, fix installed